pkg/operator/configobservation/configobservercontroller/observe_config_controller.go

```diff
@@ -31,7 +31,7 @@ func NewConfigObserver(
 	interestingNamespaces := []string{
 		operatorclient.GlobalUserSpecifiedConfigNamespace,
 		operatorclient.GlobalMachineSpecifiedConfigNamespace,
-		operatorclient.TargetNamespace,
+		"openshift-etcd",
```

Comment:
Just curious why not use the constant?

https://github.com/openshift/cluster-etcd-operator/blob/2ffe45d0e01405f7e7e742e0b39a3292a678eb5f/pkg/operator/operatorclient/interfaces.go#L7

Reply:
We don't generally create constants that are used in a few places and cannot change.  These four namespaces are generally the exception, but none of the resource names for instance.